icons/hammer.svg

```diff
@@ -0,0 +1,14 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+    <path d="M15 12l-8.5 8.5c-.83.83-2.17.83-3 0h0a2.12 2.12 0 010-3L12 9"/>
+    <path d="M18 56l4-4M21 53l-1.15-1.15A2.91 2.91 0 0119 49.8V49l-2.5-2.5a5.11 5.11 0 00-3.62-1.5h-2.82l.85.75a5.66 5.66 0 011.9 4.24v1.43l1.84 1.83h2.27L19 55" transform="matrix(1.0889 0 0 1.0909 -1.96 -46.1)"/>
```

Comment:
Can you remove the transform matrix?

Reply:
Maybe, it wasn't as easy as just removing the tag so but I think that there are some options in Inkscape to remove it. I'll try it later today.

Reply:
Cool!